staubli_val3_driver/val3/ros_server/ros_server.dtx

```diff
@@ -32,7 +32,7 @@
       </Value>
     </Data>
     <Data name="tTool" access="public" xsi:type="array" type="tool" size="1">
-      <Value key="0" fatherId="flange[0]" ioLink="valve1" />
+      <Value key="0" fatherId="flange[0]" ioLink="ValveIO\valve1" />
```

Comment:
Is this also for CS8(C) controllers?

Reply:
The way that valve IO is handled changed from CS8 to CS8C, so `"valve1"` would be for CS8 and `"ValveIO\valve1"` for CS8C. This doesn't seem to be used by the application (no references to `tTool.gripper` appear in the code), so it could be removed to not cause issues between different controllers.